pkg/handlers/nutanix/mutation/machinedetails/variables_test.go

```diff
@@ -43,6 +47,7 @@ func TestVariableValidation(t *testing.T) {
 								Name: &testPEClusterName,
 							},
 							MemorySize:     resource.MustParse("8Gi"),
+							Project:        testProject,
```

Comment:
given project is optional, we should separate out the test for it so that we can have one with all required and then one for each optional fields like project, additionalCategories in my opinion